### PR TITLE
update index.md

### PR DESCRIPTION
**Pull request by LokeshAggarwal1997**

### CODE REVIEW

**Comment by LokeshAggarwal1997** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
>  **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
>  **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fistio%2Fistio.io%2Fpull%2F6027) for more info**.

**Comment by LokeshAggarwal1997** [comment]
@googlebot I signed it!

**Comment by dgn** [comment]
Good catch; I wasn't aware this has changed. This differs from istio/istio, not sure that's intended; @costinm this goes back to one of your early istio/installer commits - should we backport this behavior to istio/istio?

**Comment by LokeshAggarwal1997** [comment]
@dgn please  merge this pull request, if there is any issue then tell me

**Comment by ericvn** [comment]
I'm not a fan on line#29 (which I know wasn't edited under this PR), but `Edit the `istio` configmap to enable policy checks.`  doesn't seem to me to match an `istioctl manifest apply`.  I think it might be better worded as `Apply an `istio` manifest to the cluster`.

**Comment by frankbu** [comment]
@ericvn Good point. How about change from:
```
Edit the istio configmap to enable policy checks.
```
to:
```
Update the Istio configuration to enable policy checks.
```

**Comment by adammil2000** [comment]
@LokeshAggarwal1997 Thanks for the contribution! Did you see the comment left by Frank and Eric?

**Comment by LokeshAggarwal1997** [comment]
@adammil2000, please review PR again.

File: content/en/docs/tasks/policy-enforcement/enabling-policy/index.md [hunk 1]
```diff
@@ -26,12 +26,12 @@ which enables policy checks by default.
 
     If policy enforcement is enabled (`disablePolicyChecks` is false), no further action is needed.
 
-1. Edit the `istio` configmap to enable policy checks.
+1. Update the `istio` configmap to enable policy checks.
```
Comment (frankbu) [inline content/en/docs/tasks/policy-enforcement/enabling-policy/index.md]:
```suggestion
1. Update the `istio` configuration to enable policy checks.
```

Comment (LokeshAggarwal1997) [reply content/en/docs/tasks/policy-enforcement/enabling-policy/index.md]: I just forget about it !!

Comment (LokeshAggarwal1997) [reply content/en/docs/tasks/policy-enforcement/enabling-policy/index.md]: I think, now it's ok @frankbu

**Comment by frankbu** [comment]
Lint errors are because of this unrelated change: https://github.com/istio/istio/pull/19954#issuecomment-571606204

**Pull request merged**